README.md

```diff
@@ -42,6 +42,23 @@ module "vpc" {
 
         subnet-02 = []
     }
+    
+    routes = [
+        {
+            name             = "egress-inet"
```

Comment:
Let's avoid (potentially confusing) abbreviations and call this `egress-internet`.